src/order/lattice.lean

```diff
@@ -183,6 +263,38 @@ class semilattice_inf (α : Type u) extends has_inf α, partial_order α :=
 (inf_le_right : ∀ a b : α, a ⊓ b ≤ b)
 (le_inf : ∀ a b c : α, a ≤ b → a ≤ c → a ≤ b ⊓ c)
 
+-- TODO: deduplicate this with semilattice_sup.mk' using `order_dual` / automation
```

Comment:
This should be straightforward, right?

Reply:
Maybe! I haven't used `order_dual` much so I wasn't able to figure it out.